### ISSUE TITLE
HNY <> GIV

### ISSUE THREAD

**Issue by chad-fi** [Giveth/GIVeconomy#629]
~400K liquidity pool already exists on Honeyswap:
https://info.honeyswap.org/#/pair/0x08ea9f608656a4a775ef73f5b187a2f1ae2ae10e

Suggestion is to add liquidity to either another DEX on Gnosis (concentrated) or on mainnet (thinned)

**Comment by chad-fi** [comment]
Turning off farming for existing Honeyswap pool - size expected to shrink.

Let's ask 1Hive if they are interested in mainnet liquidity to create arb opp.

**Comment by chad-fi** [comment]
Next step:
Have to prepare a proposal for GIV forum, share it with Paul that will make proposal on 1Hive forum

**Comment by chad-fi** [comment]
Either Friehiet or chadfi to make post

**Comment by chad-fi** [comment]
Draft:

## Proposal Information

tldr:
Mutually beneficial token swap to create a 50k USD equivalent shared liquidity pool of $50k GIV <> $50k HNY on Honeyswap (Gnosis Chain).

Proposal description:
I propose Giveth to use 50,000 DAI worth of GIV tokens from the Giveth Liquidity Safe (liquidity.giv.eth) to create a liquidity position on Honeyswap with 50,000 DAI worth of HNY tokens

Proposal Rationale
1Hive is one of the first community first DAOs that many Giveth followers already know.
They launched several important project in the xdai/GC ecosystem:
- Honeyswap, UniV2 fork, that at the moment has 6.5m TVL
- Celeste, arbitration system that is one of the building blocks of conviction voting
- Gardens, a no code platform to launch community first DAOs

1Hive and Giveth are already part of the Water shared liquidity project, and a further token swap can help tighten up the two communities.

## Funding Information

Amount of GIV requested: 50,000 DAI worth of GIV tokens, at current market price equivalent to 897456 GIV.

Funds shall be transferred to a Gnosis Safe
one question, for the HNY-GIV swap do we already have a safe we can use ? or we just swap the tokens and each DAO creates a liquidity pool ?

**Comment by chad-fi** [comment]
https://docs.google.com/document/d/113ta6dqhIW6HZGYwXdACjEVbrxGGIPbSQrPq5Dq-Jqc/edit

**Comment by chad-fi** [comment]
Paul from 1Hive will post the proposal soon, followed by freiheit.

**Comment by chad-fi** [comment]
1Hive needs to increase the spending limit from the common pool - on hold until they work this out.

**Comment by chad-fi** [comment]
Harvest proposal has a blocker, this needs to be worked through internally before a 15k+ pool is made. They are also concerned with inflation of GIV token, this may be a point of contention (aka sell pressure for HNY token). 

Giveth is strong, regular feature launches, inflation can be battled with GIVpower (staking/locking). FWIW HNY is also inflationary (is minted for maintenace).

Waiting for 1Hive blocker to clear, then will revisit for larger pool.
